app/views/gradebooks/speed_grader.html.erb

```diff
@@ -198,9 +200,11 @@
               <%= render partial: "grading_box_extended",
                          locals: { assignment: @assignment } %>
               <span class="turnitin_score_container"></span>
+              <span class="vericite_score_container"></span>
```

Comment:
Since turnitin and vericite are mutually exclusive, let's use the same container for both.